session-vbox/TODO

```diff
@@ -0,0 +1,11 @@
+== TODO
+
+ * Need to set bidirectional shared clipboard when creating disk image itself.
+
+== Known Issues
+
+ * Vagrant adds the default shared folder mapping, but fails to mount it.
+
+ * Packed Vagrantfile is not initialized for some reason.
+
+ * Defining no. of cpus in "Vagrantfile" does not work.
```

Comment:
We may need to set the max no. of cpus while creating the box, so that Vagrantfile can chose upto that no. for cpus.

Reply:
Even that doesn't work. But this used to work in my laptop. I guess it has to do with some host virtualization settings.